book/src/language-features/target.md

```diff
@@ -1,4 +1,4 @@
-# Query header: target dialect & version
+# Target: Dialect & Version
```

Comment:
I agree the existing title feels awkward. But `version` isn't part of the `target`, it's the version of _input_...

I guess we could revert to "Target & version"?